### PR TITLE
werft/VM: Fix certificates and lack of resources

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

Problems with certificates was that we were issuing certificates for `*.ws-dev.<branch>.preview.gitpod-dev.com`, while starting workspace under the domain `*.ws.<branch>.preview.gitpod-dev.com`.

image
image

I've spent a lot more time than I feel comfortable to admit trying to understand how to configure Gitpod to start workspaces on a different domain. I couldn't find out and in the end, I just changed our certificate to the domain that gitpod uses by default.

After fixing the certificate issues, I've also noticed that imagebuild pods were not starting due to lack of resources, so I'm also increasing it in this same PR(different commits).

image

## Related Issue(s)

Fixes https://github.com/gitpod-io/ops/issues/675

## Release Notes

```release-note
NONE
```

/werft with-vm=true

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **geropl** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @geropl` in a comment when ready.

Associated issue: *#675*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: .werft/build.ts [hunk 1]
```diff
@@ -711,7 +711,7 @@ export async function deployToDevWithHelm(deploymentConfig: DeploymentConfig, wo
 
         // trigger certificate issuing
         werft.log('certificate', "organizing a certificate for the preview environment...");
-        await issueMetaCerts(namespace, "certs", domain);
+        await issueMetaCerts(namespace, "certs", domain, false);
```
Comment (ArthurSens) [inline .werft/build.ts:714]: this is the helm installation, we can hardcode `false` here because we don't use helm and harvester at the same tim

**Review by mads-hartmann — COMMENTED** [review]
The changes look good to me, though I don't have a good overview of the various subdomains we use and their certificates.

I added the hold label as I wanted to ask what the status is if we merge the PR? Does workspaces start then? ☺️

I think it would be worth asking either the workspace team or meta about what it might take to change the subdomain so we wouldn't need to change the certificate ☺️

**Comment by ArthurSens** [comment]
> The changes look good to me, though I don't have a good overview of the various subdomains we use and their certificates.

Hopefully we can document this better once Harvester's previews are stable and we start the deletion/refactoring of legacy code.

> I added the hold label as I wanted to ask what the status is if we merge the PR? Does workspaces start then? ☺️

Unfortunately not, the DNS workspaces are getting are super strange... that's what I'm tackling now

> I think it would be worth asking either the workspace team or meta about what it might take to change the subdomain so we wouldn't need to change the certificate ☺️

I did, actually, and got no response. But I'll try asking again for that strange DNS that workspaces are getting

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by:

Associated issue: *#675*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**